nucypher/policy/models.py

```diff
@@ -104,7 +106,7 @@ def publish(self):
     @abstractmethod
     def revoke(self):
```

Comment:
I thought I removed this? Did it get added back in?

Reply:
It seems so. This is from current master:
https://github.com/nucypher/nucypher/blob/e4f70bf6abeded3fb17320c82a9c2f22e5bb4788/nucypher/policy/models.py#L105

Reply:
hmm okay